includes/class-wc-abstract-google-analytics-js.php

```diff
@@ -230,7 +230,7 @@ public function get_formatted_product( WC_Product $product, $variation = false )
 			$variation = $product->get_attributes();
 		}
 
-		if ( false !== $variation ) {
+		if ( false !== $variation && is_array( $variation ) ) {
```

Comment:
I think just having `if ( is_array( $variation ) )` would be sufficient, so we can remove the `false !== $variation`

Reply:
I removed the `variation` function param completely.

Reply:
Brought back the `$variation`, so we'd use it for  non-"variation" producs